pkg/scheduler/framework/plugins/noderesources/resource_limits.go

```diff
@@ -22,9 +22,7 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/klog"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
-	"k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
```

Comment:
could we remove the named one instead? unless it's too many lines changing.

Reply:
we use nodeinfo sometimes as a variable name (nodeInfo), which is a little confusing when looking at the code.